meta-mender-qemu/conf/machine/vexpress-qemu-flash.conf

```diff
@@ -59,7 +59,7 @@ IMAGE_ROOTFS_MAXSIZE = "81920"
 # also used for generating fw_env.config but because of assumptions made in
 # meta-mender-core/recipes-bsp/u-boot/u-boot-fw-utils-mender.inc we need ship
 # our own config instead.
-MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET = "0x100000"
+MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET ?= "0x100000"
```

Comment:
Changing this from local.conf will probably make u-boot out of sync with fw_env.config